integration-tests/amazon-sam/src/test/resources/projects/configure-existing-template/pom.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
AFAIK need to be in src/it, and have an invoker.properties file to actually run the tests. See the maven plugin for an example.

Reply:
Actually I am wrong, the invoker plugin should be removed as it is not needed, but the test needs to be renamed to GenerateSamConfigTestCase so maven will run it.